src/Symfony/Component/ErrorHandler/Tests/Command/ErrorDumpCommandTest.php

```diff
@@ -0,0 +1,131 @@
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\TwigBundle\Tests\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\ErrorHandler\Command\ErrorDumpCommand;
+use Symfony\Component\ErrorHandler\ErrorRenderer\ErrorRendererInterface;
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+
+class ErrorDumpCommandTest extends TestCase
+{
+    private string $tmpDir = '';
+
+    protected function setUp(): void
+    {
+        $this->tmpDir = sys_get_temp_dir().'/error_pages';
+
+        $fs = new Filesystem();
+        $fs->remove($this->tmpDir);
+    }
+
+    public function testDumpPages()
+    {
+        $tester = $this->getCommandTester($this->getKernel(), []);
+        $tester->execute([
+            'path' => $this->tmpDir,
+        ]);
+
+        $this->assertFileExists($this->tmpDir.\DIRECTORY_SEPARATOR.'404.html');
+        $this->assertStringContainsString('Error 404', file_get_contents($this->tmpDir.\DIRECTORY_SEPARATOR.'404.html'));
+    }
+
+    public function testDumpPagesOnlyForGivenStatusCodes()
+    {
+        $fs = new Filesystem();
+        $fs->mkdir($this->tmpDir);
+        $fs->touch($this->tmpDir.\DIRECTORY_SEPARATOR.'test.html');
+
+        $tester = $this->getCommandTester($this->getKernel());
+        $tester->execute([
+            'path' => $this->tmpDir,
+            'status-codes' => ['400', '500'],
+        ]);
+
+        $this->assertFileExists($this->tmpDir.\DIRECTORY_SEPARATOR.'test.html');
+        $this->assertFileDoesNotExist($this->tmpDir.\DIRECTORY_SEPARATOR.'404.html');
+
+        $this->assertFileExists($this->tmpDir.\DIRECTORY_SEPARATOR.'400.html');
+        $this->assertStringContainsString('Error 400', file_get_contents($this->tmpDir.\DIRECTORY_SEPARATOR.'400.html'));
+    }
+
+    public function testForceRemovalPages()
+    {
+        $fs = new Filesystem();
+        $fs->mkdir($this->tmpDir);
+        $fs->touch($this->tmpDir.\DIRECTORY_SEPARATOR.'test.html');
+
+        $tester = $this->getCommandTester($this->getKernel());
+        $tester->execute([
+            'path' => $this->tmpDir,
+            '--force' => true,
+        ]);
+
+        $this->assertFileDoesNotExist($this->tmpDir.\DIRECTORY_SEPARATOR.'test.html');
+        $this->assertFileExists($this->tmpDir.\DIRECTORY_SEPARATOR.'404.html');
+    }
+
+    private function getKernel(): MockObject&KernelInterface
+    {
+        $kernel = $this->createMock(KernelInterface::class);
+        $kernel
+            ->expects($this->any())
+            ->method('getContainer')
+            ->willReturn(new Container());
+
+        $kernel
+            ->expects($this->once())
```

Comment:
Should we test it? I guess a stub is enough here as well.

Reply:
Indeed, this is no longer required, I simplified it :+1: